CMSgov/easi-app
# EASI-2459 TRB Admin - Request Edits

# EASI-2459

https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=223%3A17370

## Changes and Description

- New route for trb admin request form feedback actions: `/trb/:id/:activePage/:action`
- Paths
  - `/trb/:id/initial-request-form/request-edits`
  - `/trb/:id/initial-request-form/ready-for-consult`
- New query: `CreateTrbRequestFeedbackQuery`
- Uses `useMessage` for action submission alert feedback

### Outstanding
`<EmailRecipientsFields>` needs modifications to work in this form context. EASI-2692

## QA
- Start a trb request form
- Note the trb request uuid
- Complete the form at `/trb/:id/initial-request-form/request-edits`
- See the "view feedback" notice appear on any request form step such as `/trb/requests/:id/basic`
- See the feedback message(s) at `/trb/requests/:id/feedback`
